enclave/src/attestation.rs

```diff
@@ -310,7 +277,7 @@ pub fn get_report_from_intel(fd: c_int, quote: Vec<u8>) -> SgxResult<(String, St
 	let resp_string = String::from_utf8(plaintext.clone())
 		.sgx_error_with_log("    [Enclave] error decoding tls answer to string")?;
 
-	error!("    [Enclave] resp_string = {}", resp_string);
+	debug!("    [Enclave] resp_string = {}", resp_string);
```

Comment:
Changed from an `error` to `debug`. I guess it was an `error` to see the message, without having to fiddle with the `RUST_LOG` environment variable. However, it is quite confusing when running this code and seeing an error. I first thought I introduced a new bug with my changes.